crates/subspace-networking/src/node.rs

```diff
@@ -272,6 +273,10 @@ impl Node {
         self.shared.id
     }
 
+    pub fn dsn_connection_observer(&self) -> watch::Receiver<bool> {
```

Comment:
`Node` doesn't need `dsn_` prefix on method names, nothing in `subspace-networking` needs such prefix generally. Should probably be named something like `online_status_observer`.